igcollect/statuscode2gaphite.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import datetime
+import time
+import socket
+
+from collections import Counter
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description='Script to collect statuscodes from nginx')
+    parser.add_argument('logfile',
+                        help='path to logfile')
+    parser.add_argument('-s', '--software', default='nginx',
+                        help='which tool is using it ')
+    parser.add_argument('-t', '--time', type=int, default=3,
+                        help='where to search for the time in the log file ')
+    parser.add_argument('-x', '--position', type=int, default=8,
+                        help='where to search for the code in the log file ')
+    parser.add_argument('-p', '--prefix',
+                        default='servers.{}.software.{}'.format(
+                            socket.gethostname().replace('.', '_'),
```

Comment:
Please see other collectors. We only provide simple prefix, the correct one is to be configured by puppet.

Reply:
Is this really what you want if someone forgets to configure it right to just throw it to nginx.(etc)

Reply:
We don't forget to configure the prefix because we have Puppet helper for it.